docs/configuration/configuration.md

```diff
@@ -386,6 +390,50 @@ users with thousands of services it can be more efficient to use the Consul API
 directly which has basic support for filtering nodes (currently by node
 metadata and a single tag).
 
+### `<digitalocean_sd_config>`
+
+DigitalOcean SD configurations allow retrieving scrape targets from [DigitalOcean's](https://www.digitalocean.com/)
+Droplets API.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+* `__meta_do_id`: the id of the droplet
+* `__meta_do_name`: the name of the droplet
+* `__meta_do_image`: the image name of the droplet
+* `__meta_do_private_ipv4`: the private IPv4 of the droplet
+* `__meta_do_public_ipv4`: the public IPv4 of the droplet
+* `__meta_do_public_ipv6`: the public IPv6 of the droplet
+* `__meta_do_region`: the region of the droplet
+* `__meta_do_size`: the size of the droplet
+* `__meta_do_status`: the status of the droplet
+* `__meta_do_features`: the list of features of the droplet joined by the tag separator
+* `__meta_do_tags`: the list of tags of the target joined by the tag separator
+
+```yaml
+# The information to access the DigitalOcean API.
+[ access_token: <secret> ]
+[ access_token_file: <filename> ]
+
+# The port to scrape metrics from.
+[ port: <int> | default = 80 ]
+
+# The string by which DigitalOcean tags and features are joined into the meta labels.
+[ separator: <string> | default = , ]
+
+# An optional tag used to filter the droplets
+[ tag: <string> ]
+
+# The time after which the droplets are refreshed.
+# On large setup it might be a good idea to increase this value wrt DigitalOcean
+# rate limits.
+[ refresh_interval: <duration> | default = 60s ]
+```
+
+Note that the IP address used to scrape the targets is the private IPv4 or the
+public IPv4, in that order.
+If you want to use the IPv6, you can use the [relabel](#relabel_config)
```

Comment:
No need to document that relabelling exists.

Reply:
ack